.circleci/config.yml

```diff
@@ -10,6 +10,9 @@ jobs:
 ## Build crc
     steps:
     - checkout
+    - run: sudo defaults write com.apple.dt.Xcode IDEPackageSupportUseBuiltinSCM YES
+    - run: rm ~/.ssh/id_rsa || true
+    - run: for ip in $(dig @1.1.1.1 github.com +short); do ssh-keyscan github.com,$ip; ssh-keyscan $ip; done 2>/dev/null >> ~/.ssh/known_hosts || true
```

Comment:
Thanks! sorry for breaking this..

Reply:
No worries, i also didn't notice this until this PR